android-libkiwixbuild/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+# Sets the minimum version of CMake required to build the native library.
+set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY BOTH)
+cmake_minimum_required(VERSION 3.18.1)
+
+set(CMAKE_ANDROID_STL_TYPE llvm-libc++_static)
+# Declares and names the project.
+
+project("buildkiwix")
+
+# Creates and names a library, sets it as either STATIC
+# or SHARED, and provides the relative paths to its source code.
+# You can define multiple libraries, and CMake builds them for you.
+# Gradle automatically packages shared libraries with your APK.
+
+add_library(
+        # Sets the name of the library.
+        buildkiwix
+        # Sets the library as a shared library.
+        SHARED
+
+        # Provides a relative path to your source file(s).
```

Comment:
This looks like a copy/paste.
Doing search on internet is somehow common. But please remove those comments as you are not publishing a tutorial.

Reply:
done

Reply:
done